src/geometry/euclidean/angle/sphere.lean

```diff
@@ -109,4 +109,313 @@ begin
   exact sphere.two_zsmul_oangle_eq hs.1 hs.2.1 hs.2.2.1 hs.2.2.2 hp₂p₁ hp₂p₄ hp₃p₁ hp₃p₄
 end
 
+/-- The angle at the apex of an isosceles triangle is `π` minus twice a base angle, oriented
+angle-at-point form where the apex is given as the center of a circle. -/
+lemma sphere.oangle_eq_pi_sub_two_zsmul_oangle_center_left {s : sphere P} {p₁ p₂ : P}
```

Comment:
Since every lemma in this scope except for `cospherical.two_zsmul_oangle_eq` begins `sphere.`, how about we enter the `euclidean_geometry.sphere` namespace rather than `euclidean_geometry`?